config/environments/production.rb

```diff
@@ -56,12 +64,19 @@
   # Send deprecation notices to registered listeners.
   config.active_support.deprecation = :notify
 
-  # Disable automatic flushing of the log to improve performance.
-  # config.autoflush_log = false
-
   # Use default logging formatter so that PID and timestamp are not suppressed.
   config.log_formatter = ::Logger::Formatter.new
 
+  # Use a different logger for distributed setups.
+  # require 'syslog/logger'
+  # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new 'app-name')
```

Comment:
Can you remove all the lines that were added but commented out? I don't know what they are and if we leave them commented, then we clearly don't need them :)

Reply:
@stevecrozz this is default rails settings. I prefer to leave them as-is. Just to make diff less while upgrading.